packages/shared/src/enterprise/license.ts

```diff
@@ -677,6 +677,62 @@ export async function postResendEmailVerificationEmailToLicenseServer(
   );
 }
 
+export async function createSetupIntent(licenseKey: string) {
```

Comment:
Now that we have `packages/back-end/src/enterprise`, it probably makes sense to move these methods there since they can only be called from the back-end anyway (the front-end doesn't have access to `CLOUD_SECRET`).

If this requires a bigger refactor, don't worry about it.  We can do it later in a standalone PR since the same is true for most of this file (only works from back-end).